book/Zapp-Pipes/protocols/login_and_payment_protocol.md

```diff
@@ -0,0 +1,47 @@
+# Login and Payment Protocol
+
+This document outlines the requirements for mapping the data to determine whether login and/or payment are required to gain access to a video item.
+
+## The "Free" flag
+Previously, the flag that was used to achieve this was the "free" flag in the extensions section of the item. However, there are a few drawbacks for using the "free" flag:
+1. The flag doesn't distinguish between whether access to an item requires login or payment, and this might create some confusion. 
+2. If the item requires purchase, the "free" flag doesn't specify what entitlements are needed in order to unlock the item.
+
+The suggested protocol is meant to be used by DSPs, who want to provide this information to the app, and can be used by Login / Payment plugins to interpret the action that is needed for the item to be unlocked. 
```

Comment:
```suggestion
The protocol defined below is meant to be used by DSPs who want to provide this information to the app, and can be used by Login / Payment plugins to interpret the action that is needed for the item to be unlocked. 
```
Perhaps described is more accurate than defined. I feel that if we want it to become standard, suggested is not a strong enough word. Suggested is like recommended, but even gentler.